### ISSUE TITLE
Ambient occlusion request

### ISSUE THREAD

**Issue by mgerhardy** [floooh/voxel-test#4]
this is more like a request - i would be interested about the speed and handling of AO for this demo.

**Comment by floooh** [comment]
Good idea. There's some per-vertex ambient occlusion code, and support to pass per-voxel light-information as input data in stb_voxel_renderer but I haven't looked into this yet.

**Comment by mgerhardy** [comment]
btw. have you seen this one: https://github.com/nothings/obbg

**Comment by floooh** [comment]
Looks like I starred it at some time, but forgot about it :)

**Comment by mgerhardy** [comment]
see pr #10

**Comment by mgerhardy** [comment]
![bildschirmfoto von 2016-03-18 17 44 52](https://cloud.githubusercontent.com/assets/577713/13885111/3e56fe70-ed31-11e5-9575-5a49222fd5aa.png)

**Comment by mgerhardy** [comment]
and without:
![bildschirmfoto von 2016-03-18 17 46 02](https://cloud.githubusercontent.com/assets/577713/13885135/506d97ea-ed31-11e5-9038-fbb8b37b0467.png)
